lib/main.js

```diff
@@ -157,6 +158,18 @@ app.use(function(req, res, next) {
 // Set up 
 var assets = connect.static(paths.projectFile('assets'),
                             {maxAge:86400000}); // one day
+// Precompile JS template
+var ssjsTemplate = fs.readFileSync(paths.projectFile('assets/shiny-server.js'), 'utf-8');
```

Comment:
The hope (thought not 100% true) is that /templates would just hold the user-editable HTML templates, rather than any JS files they shouldn't touch. I'd advocate leaving it here.